saidump/SaiDump.cpp

```diff
@@ -49,22 +31,21 @@ void printUsage()
     std::cout << "    -g --dumpGraph:" << std::endl;
     std::cout << "        Dump current graph" << std::endl;
     std::cout << "    -r --rdb:" << std::endl;
-    std::cout << "        Dump by parsing the RDB JSON file, which is created by rdbtools based on Redis dump.rdb that is generated by Redis SAVE command" << std::endl;
+    std::cout << "        Dump by parsing the RDB JSON file, which is created based on Redis dump.rdb that is generated by Redis SAVE command" << std::endl;
```

Comment:
Can you update the description as we no longer use redis SAVE command.

Reply:
Addressed.